lib/experimental/fonts/font-library/class-wp-rest-font-families-controller.php

```diff
@@ -370,58 +336,72 @@ public function get_item_schema() {
+	 * Retrieves the query params for the font family collection, defaulting to the 'edit' context.
 	 *
 	 * @since 6.5.0
 	 *
-	 * @return true|WP_Error True if the request has read access, otherwise a WP_Error object.
+	 * @param array $args Optional. Additional arguments for context parameter. Default empty array.
+	 * @return array Context parameter details.
 	 */
-	public function get_font_family_permissions_check() {
-		$post_type = get_post_type_object( $this->post_type );
-
-		if ( ! current_user_can( $post_type->cap->edit_posts ) ) {
-			return new WP_Error(
-				'rest_cannot_read',
-				__( 'Sorry, you are not allowed to access font families.', 'gutenberg' ),
-				array( 'status' => rest_authorization_required_code() )
-			);
+	public function get_context_param( $args = array() ) {
+		if ( isset( $args['default'] ) ) {
+			$args['default'] = 'edit';
 		}
-
-		return true;
+		return parent::get_context_param( $args );
 	}
 
 	/**
-	 * Get the params used when creating or updating a font family.
+	 * Get the arguments used when creating or updating a font family.
 	 *
 	 * @since 6.5.0
 	 *
 	 * @return array Font family create/edit arguments.
 	 */
-	public function get_create_edit_params() {
-		$properties = $this->get_item_schema()['properties'];
-		return array(
-			'theme_json_version'   => $properties['theme_json_version'],
-			// Font family settings is stringified JSON, to work with multipart/form-data.
-			// Font families don't currently support file uploads, but may accept preview files in the future.
-			'font_family_settings' => array(
-				'description'       => __( 'font-family declaration in theme.json format, encoded as a string.', 'gutenberg' ),
-				'type'              => 'string',
-				'required'          => true,
-				'validate_callback' => array( $this, 'validate_font_family_settings' ),
-				'sanitize_callback' => array( $this, 'sanitize_font_family_settings' ),
-			),
-		);
+	public function get_endpoint_args_for_item_schema( $method = WP_REST_Server::CREATABLE ) {
+		if ( WP_REST_Server::CREATABLE === $method || WP_REST_Server::EDITABLE === $method ) {
+			$properties = $this->get_item_schema()['properties'];
+			return array(
+				'theme_json_version'   => $properties['theme_json_version'],
+				// When creating or updating, font_family_settings is stringified JSON, to work with multipart/form-data.
+				// Font families don't currently support file uploads, but may accept preview files in the future.
+				'font_family_settings' => array(
+					'description'       => __( 'font-family declaration in theme.json format, encoded as a string.', 'gutenberg' ),
+					'type'              => 'string',
+					'required'          => true,
+					'validate_callback' => array( $this, 'validate_font_family_settings' ),
+					'sanitize_callback' => array( $this, 'sanitize_font_family_settings' ),
+				),
+			);
+		}
+
+		return parent::get_endpoint_args_for_item_schema( $method );
```

Comment:
This overrides the default method for getting the allowed params when creating or editing a font face.